src/main/java/bio/terra/service/job/StairwayLoggingHooks.java

```diff
@@ -13,7 +13,7 @@
 
 public class StairwayLoggingHooks implements StairwayHook {
   private static final String FlightLogFormat =
```

Comment:
Oh, and I just noticed these don't follow java conventions.. maybe something for next time
```suggestion
  private static final String FLIGHT_LOG_FORMAT =
```